packages/legacy-client/src/client/actions/actions.test.ts

```diff
@@ -81,6 +80,10 @@ describe("Actions", () => {
         ]
       >();
 
+      expectTypeOf<typeof actions.createTask>()
```

Comment:
Not really sure if this is the best thing to do here. Using `toBeCallableWith` was the only way I could get the arrays to pass through when we had the **_incorrect_** types, unfortunately, you can't use `.not` to check that we don't regress this behavior, so I put in expect-error, which should flag an error if it actually is callable with the array.

Reply:
Yeah I think this as good as we can get for now. Once you add bulk actions support this test is going to just get deleted I guess anyhow.